.eslintrc.js

```diff
@@ -3,8 +3,14 @@
 require('@uniswap/eslint-config/load')
 
 module.exports = {
-  extends: '@uniswap/eslint-config/react',
+  extends: ['@uniswap/eslint-config/react'],
   overrides: [
+    {
+      files: ['**/*.config.*', '**/*.d.ts'],
+      rules: {
+        'import/no-unused-modules': 'off',
```

Comment:
are you sure we should turn this off vs just comment things?

Reply:
yes: for config and typings files, it is typical to export things that are used outside of the transpiled package (eg not captured by the tsconfig), so I think it makes more sense to just turn this off entirely.